src/ptex.imageio/ptexinput.cpp

```diff
@@ -105,7 +105,7 @@ PtexInput::open(const std::string& name, ImageSpec& newspec)
             m_ptex->release();
             m_ptex = NULL;
         }
-        errorf("%s", perr);
+        errorf("%s", perr.c_str());
```

Comment:
What's warning did this one generate?

Reply:
It produces this error: undefined reference to `Ptex::v2_3::operator<<(std::ostream&, Ptex::v2_3::String const&)`